toonz/sources/tnztools/stylepickertool.cpp

```diff
@@ -213,6 +213,17 @@ void StylePickerTool::mouseMove(const TPointD &pos, const TMouseEvent &e) {
 
 int StylePickerTool::getCursorId() const {
   int ret;
+
+  TImageP img;
+  if (!m_active)
+    img = touchImage();
```

Comment:
This line seems to cause creating a new level when clicking an empty cell in Xsheet.

Reply:
Correcting.